addons/calendar/migrations/9.0.1.0/noupdate_changes.xml

```diff
@@ -0,0 +1,360 @@
+                                                        <div style="height:auto; width: 120px; background : #CCCCCC; font-family: Lucida Grande', Ubuntu, Arial, Verdana, sans-serif;">
+                                                            Duration
+                                                        </div>
+                                                    % endif
+                                                </td>
+                                                <td colspan="3" style="vertical-align:text-top;">
+                                                    % if not object.event_id.allday and object.event_id.duration:
+                                                        <div style="font-family: Lucida Grande', Ubuntu, Arial, Verdana, sans-serif;">
+                                                            : ${('%dH%02d' % (object.event_id.duration,(object.event_id.duration*60)%60))}
+                                                        </div>
+                                                    % endif
+                                                </td>
+                                            </tr>
+                                        <tr style=" height: 30px;">
+                                            <td style="height: 25px;width: 120px; background : # CCCCCC; font-family: Lucida Grande', Ubuntu, Arial, Verdana, sans-serif;">
+                                                <div>
+                                                    Attendees
+                                                </div>
+                                            </td>
+                                            <td colspan="3">
+                                               :
+                                                % for attendee in object.event_id.attendee_ids:
+                                                    <div style="display:inline-block; border-radius: 50%; width:10px; height:10px;background:${'color' in ctx and ctx['color'][attendee.state] or 'white'};"></div>
+                                                    % if attendee.cn != object.cn:
+                                                        <span style="margin-left:5px">${attendee.cn}</span>
+                                                    % else:
+                                                        <span style="margin-left:5px">You</span>
+                                                    % endif
+                                                % endfor
+                                            </td>
+                                        </tr>
+                                    </table>
+                                </td>
+                            </tr>
+                        </table>
+                    </div>
+                </div>
+                ]]></field>
+    </record>
+    <record id="alarm_notif_5" model="calendar.alarm">
```

Comment:
I'm not sure we should change these values, as users could customize this previously (and they are not data that can fail).

Reply:
That means removing that?

Reply:
That's it. Maybe you can include them commented and include a text clarifying it for having future references and not saying "Why I didn't include these records?"